source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,749 @@
+
+.. question::
+
+   Open the ANCOM visualizations for the donor and genotype and the taxonomy visualization artifact.
+
+   1. Are there more differentially abundant features between the donors or the mouse genotype? Did you expect this result based on the beta diversity?
+   2. Are there any features that are differentially abundant in both the donors and by genotype?
+   3. How many differentially abundant features are there between the two genotypes? Using the percentile abundances as a guide, can you tell if they are more abundant in wild type or susceptible mice?
+   4. Use taxonomy metadata visualization and search sequence identifiers for the significantly different features by genotype. What genera do they belong to?
+
+
+Longitudinal Analysis
+=====================
+
+This study includes a longitudinal component; samples from each mouse were collected 7, 14, 21, and 49 days post fecal transplant. We can use the ``q2-longitudinal`` plug-in to explore the hypothesis that a mouse’s genetic background affected the change in the microbial community of each mouse. For this longitudinal analysis, we’re going to focus on beta diversity. Alpha diversity changes wildly in infants, but it’s often stable in adults over short time periods. We’re dealing with an adult fecal community over a relatively short time period, and there is no difference in alpha diversity with time. The :doc:`longitudinal analysis tutorial <longitudinal>` is an excellent resource for exploring changes samples.
+
+PCoA-based analyses
+-------------------
+
+We can start by exploring temporal change in the PCoA using the animations tab.
+
+.. question::
+
+   Open the unweighted UniFrac emperor plot and color the samples by mouse id. Click on the “animations” tab and animate using the ``day_post_transplant`` as your gradient and ``mouse_id`` as your trajectory. Do you observe any clear temporal trends based on the PCoA?
+
+   What happens if you color by ``day_post_transplant``? Do you see a difference based on the day? *Hint: Trying changing the colormap to a sequential colormap like viridis.*
+
+
+Sometimes, it can also be useful to view the PCoA using a custom axis. Let’s use ``q2-emperor`` to make a PCoA where we can look at the time after transplant as a custom axis using the ``--p-custom-axes`` parameter.
+
+.. command-block::
+
+   qiime emperor plot \
+     --i-pcoa ./core-metrics-results/unweighted_unifrac_pcoa_results.qza \
+     --m-metadata-file ./metadata.tsv \
+     --p-custom-axes days_post_transplant \
+     --o-visualization ./core-metrics-results/unweighted_unifrac_emperor_time_axis.qzv
+
+.. included to recapitulate the step from moving pictures. Im not sure it adds much, and it covered there?
```

Comment:
I don't think this is particularly useful, and usually isn't that interesting in workshops, I would be fine with dropping the custom axis example.

Reply:
Good. Let's drop it, then. I think volatility plots are better and moving pictures covers it if people need it.